diff --git a/src/app-window.vala b/src/app-window.vala
index 005723b..9975b1a 100644
--- a/src/app-window.vala
+++ b/src/app-window.vala
@@ -431,7 +431,7 @@ private class Boxes.AppWindow: Gtk.ApplicationWindow, Boxes.UI {

             return true;
         } else if (event.keyval == Gdk.Key.A &&
-                   (event.state & default_modifiers) == Gdk.ModifierType.CONTROL_MASK | Gdk.ModifierType.SHIFT_MASK) {
+                   (event.state & default_modifiers) == (Gdk.ModifierType.CONTROL_MASK | Gdk.ModifierType.SHIFT_MASK)) {
             foreach_view ((view) => { view.unselect_all (); });

             return true;
